mattermost-plugin/server/manifest.go

```diff
@@ -45,7 +45,8 @@ const manifestStr = `
         "type": "bool",
         "help_text": "This allows board editors to share boards that can be accessed by anyone with the link.",
         "placeholder": "",
-        "default": false
+        "default": false,
+        "hosting": ""
```

Comment:
this change was generated automatically while setting up the plugin in my env, not sure if it needs to be version controlled

Reply:
The same thing happens for me as well. I've just been ignoring it, but I don't know if it's something that should actually be checked in

Reply:
Not sure what causes the back and forth on this line...but don't worry about it.